jellyfin-rpc-cli/src/main.rs

```diff
@@ -66,14 +67,18 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     #[cfg(feature = "updates")]
     updates::checker();
 
-    let conf = Config::builder()
-        .load(
-            &args
-                .config
-                .unwrap_or(get_config_path().expect("default config path couldn't be determined")),
-        )
-        .expect("config not found")
-        .build();
+    let conf_path = &args
+        .config
+        .unwrap_or(get_config_path().expect("default config path couldn't be determined"));
+
+    let conf = match Config::builder().load(conf_path) {
+        Ok(file) => file.build(),
+        Err(_error) => {
+            error!("Config file could not be found at path: {}", conf_path.red());
+            error!("Please create a proper config file: {}", "https://github.com/Radiicall/jellyfin-rpc/wiki/Setup".green());
+            std::process::exit(1)
+        }
```

Comment:
Only thing wrong i can see is that you arent printing the actual error, file permissions for instance is another thing that can cause this to error.
Apart from that lgtm

Reply:
Because the error can be related to file permissions it should also probably print this instead
```diff
-            error!("Config file could not be found at path: {}", conf_path.red());
+            error!("Config file could not be loaded from path: {}", conf_path.red());
```

Reply:
These changes have been made and an example error output is now:

```bash
> ./jellyfin-rpc 
2024-10-24 18:59:22 INFO  [jellyfin_rpc] Initializing Jellyfin-RPC
2024-10-24 18:59:23 ERROR [jellyfin_rpc] Config file could not be loaded at path: /Users/hmorin/.config/jellyfin-rpc/main.json
2024-10-24 18:59:23 ERROR [jellyfin_rpc] No such file or directory (os error 2)
2024-10-24 18:59:23 ERROR [jellyfin_rpc] Please create a proper config file: https://github.com/Radiicall/jellyfin-rpc/wiki/Setup
```